ellocate.el

```diff
@@ -35,12 +35,19 @@
 (require 's)
 (require 'f)
 (require 'ivy)
+(require 'cl-lib)
 
-(defcustom ellocate-scan-dirs '(("~/" "~/ellocate-home-db"))
```

Comment:
Notice I removed unwrapped the list here -- I think you had a list containing a list, instead of a list of strings, but correct me if I'm wrong.